CMakeLists.txt

```diff
@@ -0,0 +1,119 @@
+# Copyright 2022
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or https://www.opensource.org/licenses/mit-license.php.
+
+cmake_minimum_required(VERSION 3.1)
+if(CMAKE_VERSION VERSION_GREATER 3.14)
+  cmake_policy(SET CMP0092 NEW)
+endif()
+project(secp256k1 VERSION 0.1.0 LANGUAGES C)
+
+set(CMAKE_C_STANDARD 90)
+set(CMAKE_C_EXTENSIONS OFF)
+
+option(ENABLE_DEV_MODE "enable all binaries and modules by default but individual options can still be overridden explicitly" OFF)
+
+option(BUILD_BENCHMARK "build benchmarks" ON)
+option(BUILD_TESTS "build tests" ON)
+option(BUILD_EXHAUSTIVE_TESTS "build exhaustive tests" ON)
+option(BUILD_EXAMPLES "build examples" ${ENABLE_DEV_MODE})
+
+option(ENABLE_MODULE_ECDH "enable ECDH module" ${ENABLE_DEV_MODE})
+option(ENABLE_MODULE_RECOVERY "enable ECDSA pubkey recovery module" ${ENABLE_DEV_MODE})
+option(ENABLE_MODULE_EXTRAKEYS "enable extrakeys module" ${ENABLE_DEV_MODE})
+option(ENABLE_MODULE_SCHNORRSIG "enable schnorrsig module" ${ENABLE_DEV_MODE})
+
+option(ALLOW_EXPERIMENTAL "allow experimental configure options" ${ENABLE_DEV_MODE})
+option(USE_EXTERNAL_DEFAULT_CALLBACKS "enable external default callback functions" OFF)
+option(COVERAGE "enable compiler flags to support coverage analysis" ${ENABLE_DEV_MODE})
```

Comment:
`COVERAGE` shouldn't be enabled by `ENABLE_DEV_MODE`

Reply:
Thanks! [Fixed](https://github.com/bitcoin-core/secp256k1/pull/1113#issuecomment-1171325257).